### ISSUE TITLE
How to insert html in the text property?

### ISSUE THREAD

**Issue by Archakov06** [tommoor/react-emoji-render#8]
...

**Comment by tommoor** [comment]
Hey, could you let me know some more detail on what you're trying to achieve?

**Comment by Archakov06** [comment]
I have a function that translates new lines into <br> and converts the links into clickable. Your component escaping all HTML elements.

```
parseMessage(text){
    text = text.replace(/\b((?:https?|ftp):\/\/.[a-zР°-СЏ0-9-+&@#\/%?=~_|!:,.;]*[a-zР°-СЏ0-9-+&@#\/%=~_|])/gim,'<a href="$1" target="_blank">$1</a>');
    text = text.replace(/\n/g,'<br/>');
    return text;
  }
```

Can I somehow use your component <Emojione /> to insert text with HTML elements?

`<Emojione dangerouslySetInnerHTML={{ __html: "Hello :smile: <br/> World!" }} />` ?

**Comment by tommoor** [comment]
I just published `0.4.0` which exposes the internals as [toArray](https://github.com/tommoor/react-emoji-render#array-output). With that you can loop through the output and do further processing on the text items. The whole point of this module is to avoid using `dangerouslySetInnerHTML` though, so you should avoid that.
